cli/azd/pkg/tools/github/github.go

```diff
@@ -223,12 +257,65 @@ func (cli *ghCli) GitHubActionsExists(ctx context.Context, repoSlug string) (boo
 	return true, nil
 }
 
+func (cli *ghCli) ForceConfigureAuth(authMode AuthTokenSource) {
+	if authMode == TokenSourceFile {
+		for _, tokenEnvVarName := range TokenEnvVars {
+			cli.overrideTokenEnv = append(cli.overrideTokenEnv, fmt.Sprintf("%v=", tokenEnvVarName))
+		}
+	} else { // authMode == TokenSourceEnvVar
+		// GitHub CLI will always use environment variables first.
+		// Therefore, we simply need to clear our environment context override to force environment variable usage.
+		cli.overrideTokenEnv = nil
+	}
```

Comment:
```suggestion
	if authMode == TokenSourceFile {
	        for _, tokenEnvVarName := range TokenEnvVars {
		        cli.overrideTokenEnv = append(cli.overrideTokenEnv, fmt.Sprintf("%v=", tokenEnvVarName))
	        }
		return
	}
	// GitHub CLI will always use environment variables first.
	// Therefore, we simply need to clear our environment context override to force environment variable usage.
	cli.overrideTokenEnv = nil	
```

Reply:
This is just a ni.  Feel free to ignore..  It is just me that I don't like the `else` and I'd would try to avoid it if possible

Reply:
Conceptually, I think pattern-matching is what we want here. It's not a validation with early exists..